celery_batches/__init__.py

```diff
@@ -355,10 +374,17 @@ def flush(self, requests: Collection[Request]) -> Any:
         def on_accepted(pid: int, time_accepted: float) -> None:
             for req in acks_early:
                 req.acknowledge()
+            for request in requests:
+                request.send_event("task-started")
```

Comment:
The previous PR (#34) sets the `req.start_time` for each first. Any idea if this is necessary?

Reply:
I believe, for what we are trying to implement now, this `time_start` is not necessary for now. It will surely come handy later if we implement all events for monitoring.

Reply:
OK, we can probably start w/o it. I'll need to double check the celery code around here so we stay similar-ish.

Reply:
Fine :) I'll wait for the push then and continue working on other events such as failed/revoked/retry (where failed and retry would be the most important for application monitoring)

Reply:
>  I'll wait for the push then and continue working on other events

Sorry -- I'm unsure what you mean by this. Are you waiting on me for something?

Reply:
> > I'll wait for the push then and continue working on other events
> 
> Sorry -- I'm unsure what you mean by this. Are you waiting on me for something?

No do not worry, that would be an other topic/PR (related to our latest messages in the issue 83)

Reply:
OK! Do you see this ready for review then?

Reply:
I do !